src/ModernHttpClient/Android/OkHttpNetworkHandler.cs

```diff
@@ -12,245 +12,255 @@
 
 namespace ModernHttpClient
 {
-    public class NativeMessageHandler : HttpClientHandler

```

Comment:
Review code formatting, please. Because I can't understand what was changed in an easy way.

Reply:
it's refactoring, clear goto operators.

Reply:
I'm talking about spacing in file http://prntscr.com/gfapgs. When your VS configured differently from the repository, and for example, you use spaces instead of tabs or 4 spaces instead of 2, merging says that you changed the whole file, but was changed only a few lines. And it is hard to understand what lines were changed. In this case, necessary read the whole file to understand what was changed instead of review only a few lines.

Reply:
Okey, sorry. On SourceTree it's not necessary, invisible

Reply:
If you have time and desire, please change formatting and resend pull-request)

Reply:
Ok